code/core/src/core-server/presets/common-preset.ts

```diff
@@ -104,8 +104,32 @@ export const favicon = async (
 
 export const babel = async (_: unknown, options: Options) => {
   const { presets } = options;
-
-  return presets.apply('babelDefault', {}, options);
+  const babelDefault = await presets.apply('babelDefault', {}, options);
+  return {
+    ...babelDefault,
+    // This override makes sure that we will never transpile babel further down then the browsers that storybook supports.
+    // This is needed to support the mount property of the context described here:
+    // https://storybook.js.org/docs/writing-tests/interaction-testing#run-code-before-each-test
+    overrides: [
+      {
+        include: /(story|stories)\.[cm]?[jt]sx?$/,
```

Comment:
If you want to be really exact you probably want to:
1. Add a dot at the beginning, to not match `the-cool-stories.js`
2. Don't match on `story`, Storybook doesn't support that even if a user's glob pattern would match it. (Unless something changes recently in the default CSF Indexer that I'm unaware of)

Reply:
Good point! 

Regarding 2, do you know @shilman ?

Reply:
Looks like we do support both suffixes in our indexer and the "." bug exists there too:

https://github.com/storybookjs/storybook/blob/next/code/core/src/core-server/presets/common-preset.ts#L191-L194